src/tokenOperations/tokenOperationsFactoryConfig.ts

```diff
@@ -0,0 +1,27 @@
+import BigNumber from "bignumber.js";
+import { Address } from "../address";
+import { IAddress, IChainID, IGasLimit, IGasPrice } from "../interface";
+
+export class TokenOperationsFactoryConfig {
+    chainID: IChainID;
+    minGasPrice: IGasPrice = 1000000000;
+    minGasLimit = 50000;
+    gasLimitPerByte = 1500;
+    gasLimitIssue: IGasLimit = 60000000;
+    gasLimitESDTLocalMint: IGasLimit = 300000;
+    gasLimitESDTLocalBurn: IGasLimit = 300000;
+    gasLimitSetSpecialRole: IGasLimit = 60000000;
+    gasLimitPausing: IGasLimit = 60000000;
+    gasLimitFreezing: IGasLimit = 60000000;
+    gasLimitESDTNFTCreate: IGasLimit = 3000000;
```

Comment:
For most of these operations, the docs specifies gas limit amounts much larger than the one listed here:

https://github.com/multiversx/mx-chain-mainnet-config/blob/master/gasSchedules/gasScheduleV7.toml